test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/TemporalAccessorTest.scala

```diff
@@ -7,8 +7,8 @@ import org.junit.Test
 import org.junit.Assert._
 import org.scalajs.testsuite.utils.AssertThrows._
 
-abstract class TemporalAccessorTest {
-  val samples: Seq[TemporalAccessor]
+abstract class TemporalAccessorTest[A <: TemporalAccessor] {
+  val samples: Seq[A]
```

Comment:
Oh, now I see. Then it is fine. You could give it a more meaningful name than `A` like `TA` or `TempAcc`. 

You can also go ahead and squash the commits.